Wrappers/Dials/Symmetry.py

```diff
@@ -71,14 +71,6 @@ def set_mode_absences_only(self):
             self._laue_group = None
             self._sys_abs_check = True
 
-        def set_mode_laue_only(self):
```

Comment:
Although these are not currently used, probably useful to keep around for future use

Reply:
If these are really needed in the future wouldn't it make more sense to add them back in as properties instead of getter/setter methods?